controllers/api/v1/report_test.go

```diff
@@ -0,0 +1,113 @@
+package apiv1controllers_test
+
+import (
+	"fmt"
+	"net/http"
+
+	"go-crawler-challenge/models"
+	. "go-crawler-challenge/tests"
+	. "go-crawler-challenge/tests/custom_matchers"
+	. "go-crawler-challenge/tests/fixtures"
+
+	"github.com/bxcodec/faker/v3"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("ReportController", func() {
+	AfterEach(func() {
+		TruncateTable("oauth2_tokens")
+		TruncateTable("oauth2_clients")
+		TruncateTable("link")
+		TruncateTable("page")
+		TruncateTable("position")
+		TruncateTable("keyword")
+		TruncateTable("user")
+	})
+
+	Describe("GET /api/v1/report/:keyword_id", func() {
+		Context("given a valid access token", func() {
+			Context("given a valid report", func() {
```

Comment:
Should it be "given a valid keyword ID" as we're requesting the report of a keyword?